src/qtiRunner/core/Renderer.js

```diff
@@ -544,18 +544,18 @@ const Renderer = function(options) {
                 requiredClasses = _.intersection(requiredClasses, _renderableClasses);
 
                 //add dependencies
-                _.each(requiredClasses, function(reqClass) {
+                _.each(requiredClasses, function (reqClass) {
                     const deps = _dependencies[reqClass];
                     if (deps) {
                         requiredClasses = _.union(requiredClasses, deps);
                     }
                 });
 
-                _.forEach(requiredClasses, function(qtiClass) {
+                _.forEach(requiredClasses, function (qtiClass) {
                     let requiredSubClasses;
                     if (_renderableSubclasses[qtiClass]) {
                         requiredSubClasses = _.intersection(requiredClasses, _renderableSubclasses[qtiClass]);
-                        _.each(requiredSubClasses, function(subclass) {
+                        _.each(requiredSubClasses, function (subclass) {
```

Comment:
https://lodash.com/docs/4.17.15#forEach
```suggestion
                        _.forEach(requiredSubClasses, function (subclass) {
```